src/app/(app)/attendance/actions/create-attendance.ts

```diff
@@ -17,6 +19,8 @@ export const createAttendance = async (values: AttendanceFormValues) => {
   const { error } = await supabase.from(Tables.Attendance).insert({
     ...payload,
     status: AttendanceStatus.Pending,
+    role:loginUser.role,
```

Comment:
fix this status and role can't be  always `loginUser`  because loginUser can be incharge or shift_incharge and they can submit attendance for there shift members.